arch/lkl/include/asm/string.h

```diff
@@ -7,70 +7,35 @@
 
 /* use __mem* names to avoid conflict with KASAN's mem* functions. */
 
+#ifdef CONFIG_LKL_HOST_MEMCPY
 #define __HAVE_ARCH_MEMCPY
 extern void *memcpy(void *dest, const void *src, size_t count);
```

Comment:
I wonder if need this declaration here of memcpy function provided that we define memcpy macro below? If not, may be delete the declaration of the function alltogether? 

The same is applicable for memset and memmove functions.

Reply:
Ah, I see it is actually used in arch/lkl/lib/string.c.